hack/patches/006-version.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/vendor/knative.dev/pkg/environment/client_config.go b/vendor/knative.dev/pkg/environment/client_config.go
+index 04d4220b..d094bf0c 100644
+--- a/vendor/knative.dev/pkg/environment/client_config.go
++++ b/vendor/knative.dev/pkg/environment/client_config.go
+@@ -42,9 +42,10 @@ func (c *ClientConfig) InitFlags(fs *flag.FlagSet) {
+ 	fs.StringVar(&c.ServerURL, "server", "",
+ 		"The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+
+-	fs.StringVar(&c.Kubeconfig, "kubeconfig", os.Getenv("KUBECONFIG"),
+-		"Path to a kubeconfig. Only required if out-of-cluster.")
+-
++	if fs.Lookup("kubeconfig") == nil {
```

Comment:
This for double vendoring (https://github.com/kubernetes-sigs/controller-runtime/issues/8780), you need this so controller-runtime plays well. Otherwise you get an issue with the flag.